molecule/default/Dockerfile.j2

```diff
@@ -6,7 +6,7 @@ RUN if [ $(command -v apt-get) ]; then apt-get update && apt-get upgrade -y && a
     elif [ $(command -v dnf) ]; then dnf makecache && dnf --assumeyes install python sudo python-devel python2-dnf bash && dnf clean all; \
     elif [ $(command -v yum) ]; then yum makecache fast && yum update -y && yum install -y python sudo yum-plugin-ovl bash && sed -i 's/plugins=0/plugins=1/g' /etc/yum.conf && yum clean all; \
     elif [ $(command -v zypper) ]; then zypper refresh && zypper update -y && zypper install -y python sudo bash python-xml && zypper clean -a; \
-    elif [ $(command -v apk) ]; then apk update && apk add --no-cache python sudo bash ca-certificates shadow; fi
+    elif [ $(command -v apk) ]; then apk update && apk add --no-cache python sudo bash ca-certificates shadow dpkg; fi
```

Comment:
This is for molecule testing. Instead, since `dpgk` is a dependency for a feature (not for running Ansible), it should be moved to `var/Alpine.yml`.